### DISCUSSION TITLE
Login err with token e space-cli

### DISCUSSION THREAD

**Discussion by clecio81** [deta/space-docs#46] [Q&A]
after enter the token login, the following error appears: 

! Failed to validate access token: Get "https://deta.space/api/v0/space": dial tcp: lookup deta.space on [::1]:53: read udp [::1]:33611->[::1]:53: read: connection refused

**Comment by sudo-subg** [comment]
which OS?

**Reply by clecio81** [reply]
Linux

**Reply by sudo-subg** [reply]
this seems like an issue with DNS resolution on your Linux system.
you can try creating a file called `space_tokens` in `$HOME/.detaspace/` and in that file add the following;
```
{"access_token":"your_access_token"}
```
> ⚠️ Replace `your_access_token` with your newly generated Deta Space `access token`.

**Reply by clecio81** [reply]
this worked, but when I enter the space new code it returns this error:

`Error: Post "https://deta.space/api/v0/apps": dial tcp: lookup deta.space on [::1]:53: read udp [::1]:47120->[::1]:53: read: connection refused`
